### PR TITLE
Updates to Overview of Components doc to remove references to hot, warm, and cold paths

### PR DESCRIPTION
**Pull request by JohnPWSharp**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@JohnPWSharp__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: docs/overview-of-architecture-and-components.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ The sample solution code (available [here](https://github.com/mspnp/iot-journey/
 
 The data for all devices is directed towards a Windows Azure Event Hub. From here, the data can take several paths:
```
Comment (bennage) [inline docs/overview-of-architecture-and-components.md]: Remove "Windows"

File: docs/overview-of-architecture-and-components.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ The sample solution code (available [here](https://github.com/mspnp/iot-journey/
...
 
-2. The Stream Analytics job can also combine the data with reference information that maps device IDs to buildings. The reference information is held in Bloc storage. The combined data is then processed to generate a rolling average of the temperatures recorded by all devices in a building during the last 5 seconds. The results are stored in an Azure SQL Database. This is the *warm path*; it adds value to the raw data by merging and summarizing it. Operations staff can query this summary data in the database to view the average temperature in each building.
+2. The Stream Analytics job can also combine the data with reference information that maps device IDs to buildings. The reference information is held in Bloc storage. The combined data is then processed to generate a rolling average of the temperatures recorded by all devices in a building during the last 5 seconds. The results are stored in an Azure SQL Database. This path adds value to the raw data by merging and summarizing it. Operations staff can query this summary data in the database to view the average temperature in each building.
```
Comment (bennage) [inline docs/overview-of-architecture-and-components.md]: Bloc - > blob

File: docs/overview-of-architecture-and-components.md [hunk 3]
```diff
@@ -10,11 +10,11 @@ The sample solution code (available [here](https://github.com/mspnp/iot-journey/
...
+3. The data can queried by using the Cold Storage Processor. This is a separate application that can be run by operations staff. The Cold Storage Processor listens for temperature events as they arrive at the Event Hub (the Event Hub can also store a history of events for a configurable period, and these events will be passed to the Cold Storage Processor if it hasn't been run recently.) The Cold Storage processor copies the data for each temperature to Blob storage from where it can be subjected to detailed analysis and ad-hoc processing. Another Hive query is available to enable operations staff to summarize this data and generate a report.
```
Comment (bennage) [inline docs/overview-of-architecture-and-components.md]: The statement is not correct. The `Cold Storage Processor` reads all of the events from an event hub and writes them to blob storage. It sames the same purpose as the Stream Analytics job described in 1. The are competing implementation that we intend to compare.

File: docs/overview-of-architecture-and-components.md [hunk 4]
```diff
@@ -10,11 +10,11 @@ The sample solution code (available [here](https://github.com/mspnp/iot-journey/
 
 The data for all devices is directed towards a Windows Azure Event Hub. From here, the data can take several paths:
 
-1. The data can be used as input to a Azure Stream Analytics job which simply passes the data through to Blob storage, recording it in JSON format. This is the *hot path* that provides the least friction for the raw data, allowing it to be stored very quickly for later analysis. Operations staff are provided with a Hive query that they can run on an HDInsight cluster to take this data and generate a report listing each device and the number of events it generated.
+1. The data can be used as input to a Azure Stream Analytics job which simply passes the data through to Blob storage, recording it in JSON format. This path provides the least friction for the raw data, allowing it to be stored very quickly for later analysis. Operations staff are provided with a Hive query that they can run on an HDInsight cluster to take this data and generate a report listing each device and the number of events it generated.
```
Comment (bennage) [inline docs/overview-of-architecture-and-components.md]: These events might not be "for later analysis". There are scenario where the data is simply stored for auditing and accountability.

**Comment by bennage** [comment]
There are a couple of things that I'd like to tweaks, but I'll open issues for those.

**Pull request merged**